PendingReleaseNotes

```diff
@@ -3,3 +3,41 @@
 * RGW's default backend for `rgw_enable_ops_log` changed from RADOS to file.
   The default value of `rgw_ops_log_rados` is now false, and `rgw_ops_log_file_path`
   defaults to "/var/log/ceph/ops-log-$cluster-$name.log".
+* RGW's pubsub interface now returns boolean fields using bool. Before this change,
+  `/topics/<topic-name>` returns "stored_secret" and "persistent" using a string
+  of "true" or "false" with quotes around them. After this change, these fields
+  are returned without quotes so they can be decoded as boolean values in JSON.
+  The same applies to the `is_truncated` field returned by `/subscriptions/<sub-name>`.
+* RGW's response of `Action=GetTopicAttributes&TopicArn=<topic-arn>` REST API now
+  returns `HasStoredSecret` and `Persistent` as boolen in the JSON string
```

Comment:
Should `boolen` be \`\`boolean\`\` or \`\`bool\`\`?

Reply:
i will change it to boolean, but without the quotes. as boolean or bool is not a type defined in JSON's official document . see https://www.json.org/json-en.html . `true` and `false` just literals. they are not defined as two possible values of a certain type. boolean is just "invented" here to help user to understand the represented value.